conda/recipes/libraft/meta.yaml

```
@@ -66,21 +81,34 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
         - librmm
     requirements:
       run:
         - {{ pin_subpackage('libraft-headers-only', exact=True) }}
-        - cuda-profiler-api {{ cuda_profiler_api_run_version }}
         - librmm ={{ minor_version }}
-        - libcublas {{ libcublas_run_version }}
-        - libcublas-dev {{ libcublas_run_version }}
-        - libcurand {{ libcurand_run_version }}
-        - libcurand-dev {{ libcurand_run_version }}
-        - libcusolver {{ libcusolver_run_version }}
-        - libcusolver-dev {{ libcusolver_run_version }}
-        - libcusparse {{ libcusparse_run_version }}
-        - libcusparse-dev {{ libcusparse_run_version }}
+        - cuda-version {{ cuda_spec }}
```

Pair this with the `host` `cuda-version` suggestion above.
```suggestion
        - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
```